exercises/make_it_modular/verify.js

```diff
@@ -88,6 +88,18 @@ function validateModule (modFile, callback) {
 
     callbackUsed = false
     try {
+	function compareArrays(a, b) {
+	    var a2 = (a != null) ? a.slice().sort() : new Array(),
+	    b2 = (b != null) ? b.slice().sort() : new Array(),
```

Comment:
Good idea. I've made this change. Do changes made in my branch automatically show in the merge request?